src/game_database.py

```diff
@@ -0,0 +1,179 @@
+    :param Base: This is the base model to build on top of
+    :return `object`: ORM model classes
+    """
+
+    class Metacritic(Base):
+        """
+            Creates a datamodel for the Metacritic games database to be set up
+        """
+        __tablename__ = 'metacritic'
+        id = Column(Integer, primary_key=True)
+        Game = Column(String(100), nullable=False, unique=False)
+        Genre = Column(String(100), nullable=False, unique=False)
+        Metascore = Column(Float, nullable=False, unique=False)
+        Userscore = Column(Float, nullable=False, unique=False)
+
+        def __repr__(self):
+            return '<Game {!r},MetaScore {!r},UserScore {!r}>'.format(self.Game, self.Genre, self.self.Metascore,
+                                                                      self.Userscore)
+
+    class Sales(Base):
+        """
+            Creates a datamodel for the Sales of games database to be set up
+        """
+        __tablename__ = 'sales'
+        id = Column(Integer, primary_key=True)
+        Game = Column(String(100), nullable=False, unique=False)
+        Genre = Column(String(100), nullable=False, unique=False)
+        NA_sales = Column(Float, nullable=False, unique=False)
+        EU_sales = Column(Float, nullable=False, unique=False)
+        JP_sales = Column(Float, nullable=False, unique=False)
+        Other_sales = Column(Float, nullable=False, unique=False)
+        Global_sales = Column(Float, nullable=False, unique=False)
+
+        def __repr__(self):
+            return '<Game {!r},Global_Sales {!r}>'.format(self.Game, self.Global_sales)
+
+    class Steam(Base):
+        """
+            Creates a datamodel for the Steam games by users database to be set up
+        """
+        __tablename__ = 'steam'
+        id = Column(Integer, primary_key=True)
+        Userid = Column(Integer, nullable=False, unique=False)
+        Game = Column(String(100), nullable=False, unique=False)
+        Playtime = Column(Float, nullable=False, unique=False)
+
+        def __repr__(self):
+            return '<User {!r},Game {!r}>'.format(self.userid, self.game)
+
+    class Customer(Base):
+        """
+        Creates a datamodel for the customers interacting with the app
+        """
+        __tablename__ = 'customer'
+        id = Column(Integer, primary_key=True)
+        customer_first_name = Column(String(100), nullable=False, unique=False)
+        customer_last_name = Column(String(100), nullable=False, unique=False)
+        customer_fav_game = Column(String(100), nullable=False, unique=False)
+        customer_fav_genre = Column(String(100), nullable=False, unique=False)
+        customer_time_spent = Column(Float, nullable=False, unique=False)
+        customer_rating = Column(Integer, nullable=True, unique=False)
+
+        def __repr__(self):
+            return '<User {!r},Favorite_Game {!r}>'.format(self.customer_id, self.customer_fav_game)
+
+    return Metacritic,Customer,Steam,Sales
```

Comment:
this is a massive function. would recommend shortening as per previous comment. the fausto hand rule is a good rule of thumb.

Reply:
I will look into this and cut it down.